Polyfills/Window/CMakeLists.txt

```diff
@@ -3,16 +3,31 @@ set(SOURCES
     "Source/Window.h"
     "Source/Window.cpp")
 
+if(WIN32)
```

Comment:
For platform specific files, the practice is to use the platform in the file name https://github.com/BabylonJS/BabylonNative/blob/ab4991d247ca7d74923c6924fe4258f67f8bb2df/Plugins/NativeEngine/CMakeLists.txt#L25

Reply:
Hey @CedricGuillemet, what are your thoughts on using something along the lines of:

"Polyfills/Window/Window${PLATFORM}.cpp"
vs
"Polyfills/Window/${PLATFORM}/Window.cpp" for the convention?

Reply:
What would ${Platform} evaluate to here?

---
In reply to: [565474829](https://github.com/BabylonJS/BabylonNative/pull/582#discussion_r565474829) [](ancestors = 565474829)

Reply:
I think the second one is the convention we already have elsewhere. For example: https://github.com/BabylonJS/BabylonNative/tree/master/Dependencies/UrlLib/Source

Reply:
Sounds good, I can change to use that convention.

Reply:
I have to keep the convention of each platform header sharing the same name here, since we want the same shared source to include it. But I was able to use ${BABYLON_NATIVE_PLATFORM} to handle the build flavor to minimize platform-specific ${SOURCE} declarations.